lib/Minify/Env.php

```diff
@@ -78,8 +78,7 @@ public function get($key = null)
      */
     protected function computeDocRoot(array $server)
     {
-        if (empty($server['SERVER_SOFTWARE'])
-                || 0 !== strpos($server['SERVER_SOFTWARE'], 'Microsoft-IIS/')) {
+        if (isset($server['SERVER_SOFTWARE']) && 0 !== strpos($server['SERVER_SOFTWARE'], 'Microsoft-IIS/')) {
             throw new InvalidArgumentException('DOCUMENT_ROOT is not provided and could not be computed');
```

Comment:
What if SERVER_SOFTWARE is "nginx"? Maybe we should not check this at all and only fail if SCRIPT_FILENAME or SCRIPT_NAME is missing?